airflow/providers/amazon/aws/executors/ecs/ecs_executor_config.py

```diff
@@ -0,0 +1,113 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This module mostly builds the run task kwargs from default config values, individual configs optionally set by the user, spliced with a kwargs json config optionally provided by the user. As well as setting a couple values that are required by the ECS executor. This code is a little complicated, and is hopefully well commented to annotated what's being done.